skietbaan_fe/src/components/Documents.js

```diff
@@ -1,69 +1,117 @@
 import React, { Component } from 'react';
 import '../components/Documents.css';
 import { getCookie } from './cookie.js';
+import { Collapse } from 'react-collapse';
+import {BASE_URL} from "../actions/types.js";
+import {availableIcon} from "../resources/sendDoc.png";
+import {notAvailableIcon} from "../resources/sendDoc.png";
+
 class Documents extends Component {
     constructor(props) {
         super(props);
         this.state = {
           value: "",
           value2: "",
+          collapseFilter: false,
```

Comment:
be more descriptive with these filters so collapseUserFilter for example instead of filter1 filter2